sdc/config.py

```diff
@@ -91,3 +91,8 @@
 '''
 Default value for a pointer intended to use as Numba.DefaultPassBuilder.define_nopython_pipeline() in overloaded function
 '''
+
+use_default_dataframe = distutils_util.strtobool(os.getenv('SDC_CONFIG_USE_DEFAULT_DATAFRAME', 'True'))
+'''
+Default value used to select compiler pipeline in a function decorator
```

Comment:
Description is not right - should be something like "Config variable used to select DataFrameType model (default is legacy model)"